src/Stripe.Tests.XUnit/accounts/deleting_external_account.cs

```diff
@@ -1,29 +0,0 @@
-﻿using System.Linq;
```

Comment:
Removed this because I don't think it's useful to verify the exception. Especially because it's an API decision that we will likely lift in the future